types/rn-material-ui-textfield/index.d.ts

```diff
@@ -0,0 +1,127 @@
+// Type definitions for rn-material-ui-textfield 1.0
+// Project: https://github.com/gabrieldonadel/rn-material-ui-textfield#readme
+// Definitions by: Craig Duckett <https://github.com/craigwduckett>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from 'react';
+import {
+  StyleProp,
+  TextInputProps,
+  TextStyle,
+  ViewStyle,
+  NativeSyntheticEvent,
+  TextInputFocusEventData,
+  TextInputChangeEventData,
+  ColorValue
+} from 'react-native';
+
+export interface ContentInset {
+    top?: number;
+    left?: number;
+    right?: number;
+    label?: number;
+    input?: number;
+}
+
+export interface LabelOffset {
+    /**
+     * Horizontal offset for inactive state
+     */
+    x0?: number;
+    /**
+     * Vertical offset for inactive state
+     */
+    y0?: number;
+    /**
+     * Horizontal offset for active state
+     */
+    x1?: number;
+    /**
+     * Vertical offset for active state
+     */
+    y1?: number;
+}
+
+export interface TextFieldProps extends TextInputProps {
+    textColor?:	ColorValue;
+    fontSize?: number;
+    labelFontSize?: number;
+    lineWidth?: number;
+    activeLineWidth?: number;
+    disabledLineWidth?: number;
+    tintColor?: string;
+    baseColor?: string;
```

Comment:
You should update all color-related props to use `ColorValue`
```suggestion
    tintColor?: ColorValue;
    baseColor?: ColorValue;
```